<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidSettingsWidget</class>
 <widget class="QWidget" name="AndroidSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Android Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="SDKLocationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Android SDK location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="SDKLocationLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="SDKLocationPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="sdkWarningIconLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_warning.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sdkWarningLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="NDKLocationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Android NDK location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="NDKLocationLineEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="NDKLocationPushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="ndkWarningIconLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_error.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="toolchainFoundLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="CreateKitCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Automatically create kits for Android tool chains</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="kitWarningIconLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_warning.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="kitWarningLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="AntLocationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ant location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="AntLocationLineEdit"/>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="AntLocationPushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="OpenJDKLocationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>JDK location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="OpenJDKLocationLineEdit"/>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="OpenJDKLocationPushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="AVDManagerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QPushButton" name="AVDStartPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="AVDManagerLabel">
          <property name="text">
           <string>AVD Manager</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="DataPartitionSizeLable">
          <property name="text">
           <string>System/data partition size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="DataPartitionSizeSpinBox">
          <property name="suffix">
           <string> Mb</string>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="manageAVDPushButton">
          <property name="text">
           <string>Start Android AVD Manager</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="AVDRemovePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="AVDAddPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="4">
       <widget class="QTableView" name="AVDTableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>129</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../projectexplorer/projectexplorer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>NDKLocationLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>ndkLocationEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NDKLocationPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>browseNDKLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SDKLocationLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>sdkLocationEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SDKLocationPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>browseSDKLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AntLocationLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>antLocationEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AntLocationPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>browseAntLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenJDKLocationLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>openJDKLocationEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenJDKLocationPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>browseOpenJDKLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AVDAddPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>addAVD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AVDRemovePushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>removeAVD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AVDStartPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>startAVD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AVDTableView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>avdActivated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AVDTableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>avdActivated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DataPartitionSizeSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>dataPartitionSizeEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manageAVDPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>manageAVD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CreateKitCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AndroidSettingsWidget</receiver>
   <slot>createKitToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sdkLocationEditingFinished()</slot>
  <slot>ndkLocationEditingFinished()</slot>
  <slot>browseSDKLocation()</slot>
  <slot>browseNDKLocation()</slot>
  <slot>antLocationEditingFinished()</slot>
  <slot>browseAntLocation()</slot>
  <slot>addAVD()</slot>
  <slot>removeAVD()</slot>
  <slot>startAVD()</slot>
  <slot>avdActivated(QModelIndex)</slot>
  <slot>dataPartitionSizeEditingFinished()</slot>
  <slot>openJDKLocationEditingFinished()</slot>
  <slot>browseOpenJDKLocation()</slot>
  <slot>manageAVD()</slot>
  <slot>createKitToggled()</slot>
 </slots>
</ui>
